how-to-docs/drone-how-to.md

```diff
@@ -286,11 +286,11 @@ deploy-to-prod:
 And deploy them accordingly:
 
 ```bash
-$ drone deploy UKHomeOffice/<your_repo> 15 preprod
-$ drone deploy UKHomeOffice/<your_repo> 16 prod
+$ drone deploy ukhomeoffice/<your_repo> 15 preprod
+$ drone deploy ukhomeoffice/<your_repo> 16 prod
 ```
 
-[Read more on environment and constraints](http://readme.drone.io/0.5/usage/constraints/#environment)
+Read more on [environments](http://docs.drone.io/environment/).
```

Comment:
Should probably link to the docs for the specific version of Drone we're using (v0.8, I believe).

Reply:
I think they stopped doing docs for specific versions after 0.5.

_docs.drone.io_ don't seem to have versions and _readme.drone.io_ seems to only be for v0.5 - drone/docs#257

Reply:
Ah yes, I see what you mean; it's worth mentioning that we're currently on v0.8 of Drone (OR pointing them to a page where we show version numbers for our services).

Reply:
I think we can mention the version number in the installation section (which has not been updated yet).

Reply:
Sounds good.